app/views/course/achievements/_achievement.html.slim

```diff
@@ -8,12 +8,14 @@
     div.description
       = achievement.description
 
+  td = achievement.specific_conditions.map(&:title).to_sentence
```

Comment:
I don't think we can avoid it. This part of the UI shows all conditions of an achievement, regardless of type, so we need to query every specific condition table anyway. Unless we remove this display?